### PR TITLE
added text

### PR DESCRIPTION
**Pull request by amautacelt**

### CODE REVIEW

File: demo.txt [hunk 1]
```diff
@@ -0,0 +1 @@
+adding Vlad and Mohammad
```
Comment (VDozorov) [inline demo.txt:1]: Mention that this is for the practice

**Comment by MHasan159** [comment]
add a console log.

**Review by VDozorov — CHANGES_REQUESTED** [review]

**Review by amautacelt — COMMENTED** [review]
Will change. Thanks

**Review by MHasan159 — APPROVED** [review]
resolved issue

**Review by VDozorov — APPROVED** [review]
Ok, it's fine actually

**Pull request merged**